tests/integration/cli/test_misc.py

```diff
@@ -13,10 +15,23 @@
         ["networks"],
         ["networks", "list"],
         ["plugins"],
+    ),
+)
+def test_invocation(ape_cli, runner, args):
+    result = runner.invoke(ape_cli, args)
+    assert result.exit_code == 0
+
+
+# Only run these tests once (limited to single arbitrary project).
+# This is to prevent Github from rate limiting us and causing test failures.
+@skip_projects_except(["empty-config"])
+@pytest.mark.parametrize(
+    "args",
+    (
         ["plugins", "list"],
```

Comment:
If these give us troubles down the line, we should perhaps add a `GITHUB_ACCESS_TOKEN` env var to the repo secrets or something.